mono/metadata/object.c

```diff
@@ -5381,7 +5381,6 @@ mono_object_new_common_tail (MonoObject* o, MonoClass* klass, MonoError* error)
 	return o;
 }
 
-#if 0 // FIXMEcoop awaiting https://github.com/mono/mono/pull/6876
 /**
  * mono_object_new_handle_tail:
  *
```

Comment:
Static functions don't need a mono_ prefix.

Reply:
Removed on this and the non-handle on.